recipes/pyhmc/add-license.patch

```diff
@@ -0,0 +1,50 @@
+--- /dev/null	2016-11-22 09:55:59.296008905 -0800
++++ LICENSE	2016-12-08 15:52:45.029377387 -0800
```

Comment:
Could we please get the license added to `MANIFEST.in` for future inclusions in the `sdist` when released.